pkg/controller/cluster/cluster_controller.go

```diff
@@ -50,7 +50,7 @@ func newReconciler(mgr manager.Manager, actuator Actuator) reconcile.Reconciler
 // add adds a new Controller to mgr with r as the reconcile.Reconciler
 func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Create a new controller
-	c, err := controller.New("cluster-controller", mgr, controller.Options{Reconciler: r})
+	c, err := controller.New("cluster_controller", mgr, controller.Options{Reconciler: r})
```

Comment:
Is `-` disallowed by the new controller-runtime?

Reply:
Not by controller-runtime, but opencensus uses this name to make the metrics and in the newer version dashes aren't allowed

It also matches the go file, so win-win? :D